### ISSUE TITLE
setting up take forever :(( 

### ISSUE THREAD

**Issue by newbiec355** [FlorianEagox/weeablind#8]
I have a problem, when installing "pip install -r requirements.txt" it always stops and shows --- "This is taking longer than usual. You might need to provide the dependency resolver with stricter constraints to reduce runtime. See https ://pip.pypa.io/warnings/backtracking for guidance."-- I waited 4, 5 hours but it didn't work

**Comment by shoyrock** [comment]
Yeah I've been having the same problem, it's been running for about 5hrs now I don't know maybe I'll do something wrong

**Comment by FlorianEagox** [comment]
Hello, I am aware of this and still not entirely sure what causes it. I think it's an issue with python 3.8's pip and the package pyannote-audio. If you remove that from the requirements txt and then comment out the import lines it should work. That package has been really finicky with different python packages on windows. I will try to publish a pip lock file so it will quit trying to do dependency resolution automatically. Will try to fix that in a couple days so it works properly on 3.8

**Comment by FlorianEagox** [comment]
Hello, I am aware of this and still not entirely sure what causes it. I think it's an issue with python 3.8's pip and the package pyannote-audio. If you remove that from the requirements txt and then comment out the import lines it should work. That package has been really finicky with different python packages on windows. I will try to publish a pip lock file so it will quit trying to do dependency resolution automatically. Will try to fix that in a couple days so it works properly on 3.8

**Comment by shoyrock** [comment]
I'm unable to install TTS; what version are you using

**Comment by shoyrock** [comment]
I decided to test this using Ubuntu and keep it like this may be the problem The conflict is caused by:
tts 0.15.0 depends on librosa==0.10.0.* spleeter 2.1.0 depends on librosa==0.8.0
To fix this you could try to:
﻿﻿﻿Loosen the range of package versions you've specified
﻿﻿﻿remove package versions to allow pip attempt to solve the dependency conflict.

**Comment by FlorianEagox** [comment]
Hey you, buckaroos! I think I've found a solution! In the latest commit, I've made a lock file on my system that has all the correct versions of everything, so pip doesn't go on a wacky dependency resolution quest! Coqui also updated versions, so we're on 3.10 now!

So if you do:
```
python3.10 -m venv venv
.\venv\Scripts\activate
pip install -r requirements-win310.txt --no-deps
```

We should be COOKING!
